File: rgb.py
#!/usr/bin/env python
#Author: Nazmus Nasir
#Website: https://www.EasyProgramming.net

import pigpio
import sys

red = sys.argv[1]
green = sys.argv[2]
blue = sys.argv[3]

pi = pigpio.pi()

#old 17,24,22
#replace first argument below with the GPIO Pin number associated with the specified color
pi.set_PWM_dutycycle(18,red)
pi.set_PWM_dutycycle(24,green)
pi.set_PWM_dutycycle(20,blue)

File: rgb.php
<?php
/*
	Author: Nazmus Nasir
	Website: https://www.EasyProgramming.net
*/

if(isset($_GET['red'])){
    $red = $_GET['red'];
    $green = $_GET['green'];
    $blue = $_GET['blue'];
    
    //trigger rgb.py python code with the red, green, and blue params
    $output = exec('python rgb.py ' . $red . ' ' . $green . ' ' . $blue . ' 2>&1');
    
    //ouput for troubleshooting
    echo $output . '<br />'; 
}

?>
